.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,36 +2,42 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'review'
```

Comment:
There is no review label.

Reply:
Intention was to relabel for sorting things after review to communicate the status of each issue ie information-needed, bug, feat, discussion, etc.  
Would you prefer another name or to remove?
For example #3345 needs more information, which is only visible when your check the issue.

Reply:
I would remove that for now, and move the label discussion to #2824.